src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -937,7 +944,7 @@ internal IEnumerable<NodeSearchElementViewModel> Search(string search, IEnumerab
         /// <param name="useLucene"> Temporary flag that will be used for searching using Lucene.NET </param>
         internal IEnumerable<NodeSearchElementViewModel> Search(string search, bool useLucene)
         {
-            if (useLucene)
```

Comment:
if `bool useLucene` is not used any more then the parameter in `Search(string search, bool useLucene)` should be also deleted, no?

Reply:
yes will update it.